h/activity/views.py

```diff
@@ -54,13 +54,11 @@ def eager_load_documents(query):
         timeframes = bucketing.bucket(anns)
 
         for timeframe in timeframes:
-            for document, annotations in timeframe.document_buckets.items():
-                results = []
-                for annotation in annotations:
+            for document, bucket in timeframe.document_buckets.items():
+                for index, annotation in enumerate(bucket.annotations):
```

Comment:
Yeesh. No need to fix this now, but obviously this can't be how we fetch groups for real.